docusaurus/docs/guides/alt-server-certs.md

```diff
@@ -4,68 +4,75 @@ sidebar_label: Public CA Certs
+
+Publicly verifiable server certificates are not necessary, or even desirable, for Ziti identities that are "enrolled." This is because enrolled identities are always configured to explicitly trust the Ziti controller's known CAs. Enrolled clients must verify the server certificate chains they encounter from one of the known CAs. 
 
 ## Ziti Controller
 
-The main server certificate that needs to be configured for the Ziti controller is that of the client-management API's bind point in the `web` section.
+Ziti has a conventional way of configuring [`identity`](../reference/30-configuration/controller.md#identity) certificates for the controller's TLS servers. Anywhere that you can configure a controller `identity` section you can also configure `alt_server_certs`. The controller's leaf certificates are managed externally, not by Ziti itself.
+
+The controller will select the correct server certificate to present by reading the Server Name Indication (SNI) to match a DNS Subject Alternative Name (SAN) from all available server certificates configured in that particular `identity` section. 
+
+:::note
+The controller will select the first match in the event that two certificates have the same DNS SAN. You must configure `alt_server_certs` with DNS SANs that are distinct from the `server_cert` property of the same `identity` section.
+:::
+
+The simplest way to configure the controller with alternative certificates is with the main `identity` section. This is the root identity of the control plane, often abbreviated "ctrl." 
+
+If no other `identity` sections are configured then this identity will be used whenever a client or server or CA certificate is needed.
 
 ```yaml
-web:
-  - name: client-management
-    bindPoints:
-      - interface: 0.0.0.0:1280
-        address: ctrl.example.com:1280
-    identity:
-      server_cert: /etc/letsencrypt/live/example.com/fullchain.pem
-      key: /etc/letsencrypt/live/example.com/privkey.pem
-      ca: /etc/pki/ca/example.com/certs/ca.pem  # required but unused
-      cert: /etc/pki/ca/example.com/certs/client1.pem  # required but unused
+identity:
+  cert: ctrl-plane-client.crt                  # user cert from ctrl plane CA
+  server_cert: ctrl-plane-server.crt           # server cert from ctrl plane CA with SAN "ctrl.ziti.example.com"
+  key: ctrl-plane-client.key                   # private key of client cert
+  server_key: ctrl-plane-server.key            # private key of server certs
+  ca: ctrl-plane-trust-bundle.pem              # bundle of root CAs that may be used to verify this identity's leaf cert chains
+  alt_server_certs:
+    - server_cert: lets_encrypt.cert.pem       # server cert chain from Let's Encrypt with SAN "client-pub.ziti.example.com"
+    - server_key: lets_encrypt.key.pem         # private key of alt server cert
```

Comment:
This snippet is incorrect. This is creating two invalid `alt_server_certs` one with just a cert and another with just a key. Each line prefixed w/ a dash starts a new object.

It should be:

```
  alt_server_certs:
    - server_cert: lets_encrypt.cert.pem       # server cert chain from Let's Encrypt with SAN "client-pub.ziti.example.com"
       server_key: lets_encrypt.key.pem